app/pages/utils/tools.py

```diff
@@ -56,3 +56,88 @@ def add_logo():
         unsafe_allow_html=True,
     )
 
+
+def rename_techs(label):
+    prefix_to_remove = [
+        "residential ",
+        "services ",
+        "urban ",
+        "rural ",
+        "central ",
+        "decentral ",
+    ]
+
+    rename_if_contains = [
+        "solid biomass CHP",
+        "gas CHP",
+        "gas boiler",
+        "biogas",
+        "solar thermal",
+        "air heat pump",
+        "ground heat pump",
+        "resistive heater",
+        "Fischer-Tropsch",
+    ]
+
```

Comment:
An elegant implementation per se, but looks as duplication of what we still have in the config: https://github.com/ekatef/heat-peaks-lit/blob/8231892645066dd84a6a2bcfe49d439ec2bfcae1/app/pages/utils/config.yaml#L160-L170 (those `nice names` definitions which are used in the helper scripts). 

Do you have ideas on how to revise the implementation a bit to make it more consistent? My feeling is that the functional part may remain the same, it should be enough to replace the dictionary definitions by reading from the config.

Reply:
Agree and done. With renaming, we have two goals:
1. Nicer name than used in PyPSA by default
2. Aggregate similar technologies for easier access. i.e. no need to distinct urban, rural, etc, that makes viz hard.

I hope I didn't break the other scripts with the changes...